frontend/package.json

```diff
@@ -15,6 +15,7 @@
     "@mui/lab": "^5.0.0-alpha.152",
     "@mui/material": "^5.14.17",
     "@mui/styles": "^5.14.17",
+    "@mui/system": "^5.14.17",
```

Comment:
Inside plugins/headlamp-plugin/ folder... Please run "npm run update-dependencies" to sync dependencies from frontend/ to plugins/headlamp-plugin/. This will fix the current error in CI.

Because headlamp-plugin needs to keep in sync with the frontend package versions.

Reply:
rebased and updated